cluster-autoscaler/cloudprovider/gce/mig_info_provider.go

```diff
@@ -359,6 +359,8 @@ func (c *cachingMigInfoProvider) fillMigInfoCache() error {
 				c.cache.SetMigTargetSize(zoneMigRef, zoneMig.TargetSize)
 				c.cache.SetMigBasename(zoneMigRef, zoneMig.BaseInstanceName)
 				c.cache.SetListManagedInstancesResults(zoneMigRef, zoneMig.ListManagedInstancesResults)
+				c.cache.SetMigInstancesCreatingCount(zoneMigRef, zoneMig.CurrentActions.Creating)
```

Comment:
Cluster Autoscaler has a different concept of creating and deleting than the gce instance. Please take a look at this code: https://github.com/kubernetes/autoscaler/blob/8040eadfe4f4287a14da7b3f703384c11c98dd86/cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go#L492

We need to also count instances in states other that creating and deleting.

Reply:
Updated